### PR TITLE
Update sensor.countdown_timer.markdown

### PR DESCRIPTION
**Pull request by JerryWorkman**
**Description:**
Update sensor.countdown_timer.markdown

### CODE REVIEW

File: source/_components/cover.tilt.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (Landrash) [inline source/_components/cover.tilt.markdown]: This file don't belong in this pull request.

File: source/_components/cover.tilt.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  covers:
+    front_garage_door:
+      tilt_sensor:   binary_sensor.my_tilt_switch
+      switch:        switch.my_relay_switch
+      contact_delay: 1 
+      run_time:      10
+````
+
+- **covers** (*Required*): The array that contains all tilt covers.
+  - **identifier ** (*Required*): Name of the tilt cover. Multiple entries are possible.
```
Comment (Landrash) [inline source/_components/cover.tilt.markdown]: Extra space breaking formatting after "identifier.

File: source/_components/sensor.countdown_timer.markdown [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      switch:  switch.lr_light_switch
+````
+
+- **timers** (*Required*): The array that contains all timers.
+  - **identifier ** (*Required*): Name of the timer. Multiple entries are possible.  
+    - **sensors** (*Required*): One or more binary sensor (comma separated).
+    - **switch** (*Required*): Light (or other) switch.
+    - **delay** (*Optional*): delay time. default: 20 seconds.
+    - **restart** (*Optional*): True/False, restart timer when HA is restarted. Default: False
+
```
Comment (Landrash) [inline source/_components/sensor.countdown_timer.markdown]: Part below this isn't needed in this documentation. In general documentation pages for components are minimized.

File: source/_components/sensor.countdown_timer.markdown [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+sensor countdown_timer:
+  platform: countdown_timer
+  timers:
+    lr_light:
+      sensors: binary_sensor.lr_pir1
+      switch:  switch.lr_light_switch
+````
+
+- **timers** (*Required*): The array that contains all timers.
+  - **identifier ** (*Required*): Name of the timer. Multiple entries are possible.
```
Comment (Landrash) [inline source/_components/sensor.countdown_timer.markdown]: Extra space breaking formatting after "identifier.

**Comment by JerryWorkman** [comment]
Dropping pull request for now. Will resubmit in a different form.
